tests/integration/test_script_runner.py

```diff
@@ -519,7 +519,7 @@ def pytest_parsed_arguments(request):
     return request.config.option
 
 
-@flaky(rerun_filter=delay_rerun)
+@flaky(rerun_filter=delay_rerun, max_runs=5, min_passes=1)
```

Comment:
I would consider increasing the `delay_rerun` rather than re-run it 5 times.
Right now it's `1` second. I would make the delay somewhere between `2 - 5` seconds, and maybe make the `max runs=3`

Reply:
Great idea. I'll change before merging